environment/metadata.go

```diff
@@ -134,6 +138,13 @@ func fillEKSData(e *MetaData, data *MetaDataStrings) {
 		return
 	}
 
+	eksDeploymentStrategy, ok := eksdeploymenttype.FromString(data.EksDeploymentStrategy)
+	if !ok {
+		log.Printf("Invalid deployment strategy %s. This might be because it wasn't provided for non-ECS tests", data.ComputeType)
```

Comment:
nit - this says ECS. Assuming it's a copy pasta

Reply:
Will update, will also update daemon main.tf. Forgot to add tha commit